.mdlrc

```diff
@@ -0,0 +1,2 @@
+all
+rule 'MD013', :tables => false
```

Comment:
I don't believe this is used. We remove it from the module template, so it can be deleted